.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+
+jobs:
+  python3:
+    docker:
+      - image: circleci/python:3.6.1
+    environment:
+      - MINICONDA_PATH: ~/miniconda
+      - CONDA_ENV_NAME: testenv
+      - PYTHON_VERSION: 3
+      - MATPLOTLIB_VERSION: "*"
+    steps:
+      - checkout
+      - run: ./build_tools/circle/checkout_merge_commit.sh
+      - restore_cache:
+          key: v1-datasets-{{ .Branch }}
+      - run: ./build_tools/circle/build_doc.sh
+      - save_cache:
+          key: v1-datasets-{{ .Branch }}
+          paths:
+            - ~/scikit_learn_data
+      - store_artifacts:
+          path: doc/_build/html/stable
+          destination: doc
+      - store_artifacts:
+          path: ~/log.txt
+          destination: log.txt
+      - persist_to_workspace:
+          root: doc/_build/html
+          paths: .
+
+
+  python2:
+    docker:
+      - image: circleci/python:3.6.1
```

Comment:
Please comment here that the selection of py2 happens in conda. (Do we need a python image at all?)

Reply:
I think this is simpler this way. Long story short, the minimal docker image of CircleCI does not set a UTF-8 locale and there are bugs with unicode related to that that we are trying to fix in sphinx-gallery.